packages/dds/tree/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -445,6 +523,21 @@ export function encodeValue(
 	}
 }
 
+/**
+ * Validates that incremental encoding is enabled and incrementalEncoder is.
+ * @param shouldEncodeIncrementally - Whether incremental encoding should be used.
+ * @param incrementalEncoder - The incremental encoder to use for encoding chunks.
+ */
+function validateIncrementalEncodingEnabled(
```

Comment:
This is only used once, and the place that uses it computes shouldEncodeIncrementally using `this.incrementalEncoder !== undefined` so this whole function could be replaced by a single `assert(this.incrementalEncoder !== undefined, "incremental encoding must be enabled")` in encodeIncrementalField